base/hosts_tests.c

```diff
@@ -212,6 +212,42 @@ Ensure (hosts, gvm_hosts_new_with_max_returns_error)
   assert_that (gvm_hosts_new_with_max ("127.0.0.1|127.0.0.2", 2), is_null);
 }
 
+Ensure (hosts, gvm_hosts_move_host_to_end)
+{
+  gvm_hosts_t *hosts = NULL;
+  gvm_host_t *host = NULL;
+  int totalhosts;
+  size_t current;
+
+  hosts = gvm_hosts_new ("192.168.0.0/28");
+
+  // Get first host
+  host = gvm_hosts_next (hosts);
+
+  totalhosts = gvm_hosts_count (hosts);
+  assert_that (totalhosts, is_equal_to (14));
+
+  while (g_strcmp0 (gvm_host_value_str (host), "192.168.0.9"))
```

Comment:
Just a minor nitpick. There is a `is_equal_to_string()` function for easy use. But it does not really matter.